src/txn/txn_ckpt.c

```diff
@@ -603,7 +603,7 @@ __checkpoint_stats(WT_SESSION_IMPL *session)
 
     /* Compute end-to-end timer statistics for checkpoint. */
     __wt_epoch(session, &stop);
-    msec = WT_TIMEDIFF_MS(stop, conn->ckpt_timer_scrub_end);
+    msec = WT_TIMEDIFF_MS(stop, conn->ckpt_timer_start);
```

Comment:
Thanks for fixing this. If we can add another stat for the length of scrub eviction, the pr would be more complete.